specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/examples/CheckRoutingForNumber.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "api-version": "2022-10-01-preview",
+    "endpoint": "https://resourcename.communication.azure.com/",
+    "targetNumber": "+11234567890",
+    "$select": "name"
```

Comment:
Should this also be `select` and not `$select`?

Reply:
Fixed now